smoker/logger/__init__.py

```diff
@@ -18,17 +18,39 @@
 """
 
 import logging
+import logging.config
 import logging.handlers
+import os
 from smoker.logger.level_handler import LevelHandler
 
+CONFIG_FILE = '/etc/smokerd/logging.ini'
```

Comment:
I would prefer this to be submitted on command line and passed to init() function